js_modules/dagster-ui/packages/ui-core/src/asset-graph/sidebar/AssetSidebarNode.tsx

```diff
@@ -119,9 +125,13 @@ export const AssetSidebarNode = ({
                   alignItems: 'center',
                 }}
               >
-                {isAssetNode ? <StatusDot node={node} /> : null}
-                {isGroupNode ? <Icon name="asset_group" /> : null}
-                {isLocationNode ? <Icon name="folder_open" /> : null}
+                {isAssetNode ? (
```

Comment:
I converted this to a ternary because this is another case of   `gridTemplateColumns: 'auto minmax(0, 1fr)'` making assumptions about the number of children rendered. Just seemed best to express the same expectation in this logic.